SkietbaanBE/SkietbaanBE/Controllers/ScoresController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using SkietbaanBE.Models;
+using SkietbaanBE.RequestModel;
+
+namespace SkietbaanBE.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/Scores")]
+    public class ScoresController : Controller
+    {
+        private ModelsContext _context;
+        public ScoresController(ModelsContext context)
+        {
+            _context = context;
+        }
+        [HttpGet]
```

Comment:
Why do you have two annotations beneath each other here? The Get doesn't make sense here

Reply:
forgot to remove the get sorry